### ISSUE TITLE
Bootloader issues with Ashitav3

### ISSUE THREAD

**Issue by PatricYuumi** [EdenServer/community#1679]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

**Comment by PatricYuumi** [comment]
![image](https://user-images.githubusercontent.com/50891296/58136254-70bdb780-7bfb-11e9-826a-52210c4f705e.png)

Ashita Version is 3.1.1 or whatever the latest is at the moment.

Scenario: I decided to try private server just over a week ago. Once or twice a day client would freeze and when I attempt to do anything I get the above error. If I just wait for the client to respond, it perpetually freezes over and over and is unplayable. As in literally cannot move or do anything. I just hear the game, and once every minute or so chat log will catch up.

What do I do to resolve this issue this last week? I just reboot my PC. I have a brand new PC top of line, several thousand dropped into it, no issues with any game. Yet this bootloader problem has been occurring more and more.

Today I played for about 7 hours and I did have this issue occur a few more times, but in the last 3 hours idk what happened but it got REAL bad. I've now been virutally unable to play the game for the last 3 hours due to freezing the second I login. Sometimes I get like 2 mins in and it will freeze. I've rebooted my PC over 30 times in the last 3 hours (talk about addiction?). I've reinstalled the game. I've changed my Ashita directory. I've tried several different config options in the Ashita client. I even thought I got it working again and then bam... Freeze after 2 mins :)

I've set Legacy (DX client compatible), Windows 8 compatible in Ashita properties. At a loss right now.

Let me know if you need anymore info.

**Comment by PatricYuumi** [comment]
I also turned off windows firewall and allowed pol app through windows firewall (redundant but I did both just in case)

**Comment by PatricYuumi** [comment]
I'm also not using PlayOnline. Just Ashita v3 to login 2 diff characters at the same time.

Steps:
1 - Click 'Play on Eden' on my desktop twice as administrator.
2 - This opens 2 Ashita v3 clients
3 - I then click one config, open, login, get all the way through character menu and log in as first character.
4 - I then click 2nd config, open, login, get all the way through character menu and log in as 2nd character.
5 - Try and play and then face the issue.

**Comment by PatricYuumi** [comment]
It looks like the issue is related to dualboxing as I logged in just one character and everything seems fine. So there must be something I'm doing wrong when using two characters?

**Comment by bkesecker** [comment]
Make sure you are loading one character fully at a time.  Loading simultaneously is a known issue.  Some other players have been in the same situation as you, but I have yet to hear about a resolution for them specifically.  Many players have issues that can be resolved by doing strange things.  My experience tells me Windows 8 is to blame.  Maybe try your network card(s) manufacturer's web site for updated firmware drivers.

**Comment by PatricYuumi** [comment]
I was able to get it to work, and I was also able to break it again.

I'm still playing around with what it is that causes the break and what allows it to work.

The two times I got it to work after making this thread I did similar to what you said, I logged in one character, ran around and zoned for about 10 minutes before logging in a second character.

But then I did it again and it didn't work.

My headset has also been giving me issues since this happened so I unplugged my wireless adapter for my headset and then used a aux cord instead and I was able to dual box after.

I'll do more testing when I'm home from work until I can reproduce the fix.

**Comment by PatricYuumi** [comment]
I believe it to be the headset and the firmware on it causing the issue.

I am just using a different headset is all.

This can be closed.

**Comment by PatricYuumi** [comment]
It's been about a month since I encountered this issue.

I recently just used the headset again (as it's a $300 headset) which i think may have caused the problem in the first place because I needed to use it over discord chat. I fell asleep with both characters logged in and when I woke up, no issues still. Then randomly about an hour into play both my characters dc'd. This is normal, dc's are uncommon on Eden.

I did my usual reboot desktop and log back in, I get in fine. After a minute of play same issue though. Now this RARELY happens. I tried a few more times, and low and behold it's the same issue I encountered a month ago. I've unplugged all headsets. I did a modem reset, and surprisingly this improved it for a small period, as I was able to play for an entire 10 minutes before poof, dc again, and then every minute from then on. It's been like this for about 4 hours now.

I can login solo and play, but dualboxing seems to cause the issue and yes I am logging in slowly, 1 character at a time. I've even let 1 character sit logged in for an hour before logging in an alt.

I don't mind the occasional dc's while dualboxing as for important content I will just keep 1 character offline until I'm done, but being unable to dualbox entirely causes severe disinterest in playing at all.

I'll try more troubleshooting another time but I'm reopening because the fact I was able to login for over 10 minutes after rebooting my modem made me think it could be something with my IP.

I just find it weird I'm always able to login both characters and move them fine, and then the red circle appears. It's not a full dc either. It's just, the client itself freezes sporadically and I can only do an action every 20-40 seconds, and if I try to do more than 1 action every 20-40 seconds I get the white screen with the bootloader error (shown in the original posts screenshot).

**Comment by PatricYuumi** [comment]
I would like to add I used my $300 headset for 3 days prior to encountering this issue again.

**Comment by Setz0r** [comment]
have you disabled gamepad in the ffxi config?

**Comment by PatricYuumi** [comment]
> have you disabled gamepad in the ffxi config?

I only use Ashita client before logging in, not Windower or Playonline. Where is the Gamepad FFXI config if only using Ashita?

**Comment by PatricYuumi** [comment]
Alright, I am 90% sure I fixed it. I swear, and this is so ridiculous of a fix but I'm 90% sure it's the answer and I don't know how or why....

The USB to micro-USB cable that charges my $300 headset was still plugged in during all my troubleshooting today. I had removed the wireless adapter, but as it turns out, I think it's just the USB to micro-USB charger that caused it. I've been logged in for 10 mins on both characters now and everything seems good.

If I can last 1 hour then I'll consider this resolved.
